src/onegov/agency/excel_export.py

```diff
@@ -0,0 +1,73 @@
+from collections import OrderedDict
+from io import BytesIO
+
+from onegov.agency.collections import ExtendedPersonCollection
+from onegov.agency.models import ExtendedPerson
+from xlsxwriter.workbook import Workbook
+from decimal import Decimal
+
+column_mapper = OrderedDict(
+    salutation='Anrede',
+    last_name='Nachname',
+    first_name='Vorname',
+    email='Email',
+    phone='Tel.',
+    phone_direct='Tel. direkt.',
+    born='Geburtsdatum',
+    profession='Beruf',
+    function='Funktion',
+    political_party='Partei',
+    parliamentary_group='Parlamentarische Gruppe',
+    website='Website',
+    address='Addresse',
+    notes='Notizen',
+    memberships='Mitgliedschaften'
+)
+
+
+def extract_person_data(session):
+    collection = ExtendedPersonCollection(session)
+    collection.exclude_hidden = False
+    query = collection.query().outerjoin(ExtendedPerson.memberships)
+
+    write_out = []
+
+    for person in query:
+        out_dict = OrderedDict()
+        memberships = "\n".join(
+            [f"{m.agency.title} - {m.title}" for m in person.memberships]
```

Comment:
You can remove the brackets here (`[]`), which would turn this into a generator expression - a more efficient way of solving this.